test/addons/async-hello-world/binding.cc

```diff
@@ -13,7 +20,8 @@ struct async_req {
 
 void DoAsync(uv_work_t* r) {
   async_req* req = reinterpret_cast<async_req*>(r->data);
-  sleep(1);  // Simulate CPU intensive process...
+  // Simulate CPU intensive process...
+  std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
That's right, it ships with its own copy of libc++: https://chromium.googlesource.com/chromium/buildtools/+/master/third_party/libc++/